src/api/components/source.js

```diff
@@ -471,12 +483,14 @@ module.exports = class ComponentSource extends EntitySource {
                     viewName: view.name,
                     viewPath: view.path,
                     dir: dir.path,
+                    readme: readmeFile ? new File(readmeFile) : undefined,
```

Comment:
i need to pass a `new File()` instance, otherwise there's `getContentSync()` is not a function error.

Reply:
yeah, we do need a File instance. could we reuse the `matchFile` function here? so `readme: matchFile(source.isReadme)` could maybe do the trick?